tests/mock_data/calicoctl/mesh/ipip-off/delete.yaml

```diff
@@ -4,6 +4,5 @@ metadata:
   name: ippool-1
 spec:
   cidr: 192.168.0.0/16
-  ipip:
-    mode: Off
+  ipipMode: Off
```

Comment:
Off? or Never?

Reply:
oh bugger.  as it happens i think it'll get treated as Never until we get validation in.

Reply:
but i'll fix

Reply:
Ah actually it's just in the delete file - so the Spec is irrelevant ... i'll fix though.